pkg/smokescreen/testdata/acl.yaml

```diff
@@ -15,6 +15,20 @@ services:
   - name: test-open-srv
     project: security
     action: open
+  - name: test-external-connect-proxy-blocked-srv
+    project: security
+    action: enforce
+    allowed_domains:
+      - 127.0.0.1
+    allowed_external_proxies:
+      - myproxy.com
```

Comment:
Could we add multiple external proxies to make sure the glob gets constructed correctly?

Reply:
This role is just used for the non-allowed proxies list; do you mean adding `localhost` here? How are you thinking of testing glob construction? Is just adding some different possible URL shapes helpful here, or is there something in particular you'd like us to assert against?

Reply:
All of the addresses here construct a glob which should match any `allowed_external_proxies` specific in the header. My thought was to add something like `myproxy2.com` to the end of the this and then verify it is still allowed as an external proxy in one of your tests.